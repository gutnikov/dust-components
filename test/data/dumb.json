{
  "text" : "Simple text inside",
  "authors" : [
    {
      "name" : "Alexandr",
      "surname" : "Pushkin"
    },
    {
      "name" : "Steven",
      "surname" : "King"
    }
  ]
}