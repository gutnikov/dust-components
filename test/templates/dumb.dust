<div>
	<div class="container authors my-authors">
		{#authors}
			<div class="author author-item">
				<div>{name}</div>
				<div>{surname}</div>
			</div>
		{/authors}
	</div>
</div>