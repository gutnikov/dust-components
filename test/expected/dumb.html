<div>
    <div class="container authors my-authors">
        <div class="author author-item">
            <div>Alexandr</div>
            <div>Pushkin</div>
        </div>
        <div class="author author-item">
            <div>Steven</div>
            <div>King</div>
        </div>
    </div>
</div>
